### PR TITLE
Implement full screen mode

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/6700

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/29971/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/29971/tracing/file-alice-2022-11-18-09-07-42.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/29971/tracing/file-brian-2022-11-18-09-08-05.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

File: packages/web-app-preview/src/App.vue [hunk 1]
```diff
@@ -130,6 +146,8 @@ export default defineComponent({
       isFileContentLoading: true,
       isFileContentError: false,
 
+      isFullScreenModeActivated: true,
```
Comment (kulmann) [inline packages/web-app-preview/src/App.vue:149]: Should `false` initially

File: packages/web-app-preview/src/App.vue [hunk 2]
```diff
@@ -89,6 +93,18 @@
...
+        <div class="oc-flex">
+          <oc-button
+            v-oc-tooltip="toggleFullScreenDescription"
+            class="preview-controls-fullscreen"
+            appearance="raw"
+            variation="inverse"
+            :aria-label="toggleFullScreenDescription"
+            @click="isFullScreenModeActivated = !isFullScreenModeActivated"
+          >
+            <oc-icon fill-type="line" name="fullscreen" />
```
Comment (kulmann) [inline packages/web-app-preview/src/App.vue:105]: Could you try using the icon `fullscreen-exit` when the fullscreen mode is active?

**Review by kulmann — CHANGES_REQUESTED** [review]
I don't know if `full screen` is the correct term for this... might be that the wording needs to be adjusted.
One thing that confused me when flipping though images fast, was, that the loading screen lacks the lightbox styles. i.e. you get thrown out of the (now called) full screen mode for the loading state, and then back into the full screen mode when the image has been loaded. That needs fixing, the lightbox should stay there at all times until the user goes back.

One more thing for making this great: the `ESC` key should only clear the full screen mode, but as of now it triggers the `closeApp` action instead. Could you look into that?

**Review by kulmann — APPROVED** [review]
Awesome 😎

**Pull request merged**